metals/src/main/scala/scala/meta/internal/metals/codelenses/RunTestCodeLens.scala

```diff
@@ -182,6 +190,31 @@ final class RunTestCodeLens(
     } yield new l.CodeLens(range, command, null)
   }
 
+  private def scalaCliCodeLenses(
+      textDocument: TextDocument,
+      target: BuildTargetIdentifier,
+      classes: BuildTargetClasses.Classes,
+  ): Seq[l.CodeLens] = {
+    val scriptFileName = textDocument.uri.stripSuffix(".sc")
+
+    val expectedMainClass =
+      if (scriptFileName.contains('/')) s"${scriptFileName}_sc."
+      else s"_empty_/${scriptFileName}_sc."
```

Comment:
Can you add _some_ scaladoc explaining parts like `scriptFileName.contains('/')` condition? Plain explanation in method's scaladoc, how scala scripts looks like, should be enough.